packages/services/storage/src/schema-change-model.ts

```diff
@@ -315,10 +326,31 @@ export const DirectiveRemovedModel = implement<DirectiveRemovedChange>().with({
   }),
 });
 
+export const DirectiveRepeatableAddedModel = implement<DirectiveRepeatableAddedChange>().with({
+  type: DirectiveRepeatableAddedLiteral,
+  meta: z.object({
+    directiveName: z.string(),
+  }),
+});
+
+export const DirectiveRepeatableRemovedModel = implement<DirectiveRepeatableRemovedChange>().with({
+  type: DirectiveRepeatableRemovedLiteral,
+  meta: z.object({
+    directiveName: z.string(),
+  }),
+});
+
 export const DirectiveAddedModel = implement<DirectiveAddedChange>().with({
   type: DirectiveAddedLiteral,
   meta: z.object({
     addedDirectiveName: z.string(),
+    // for backwards compatibility
+    addedDirectiveRepeatable: z.boolean().default(false) as unknown as ZodBoolean, // boolean;
```

Comment:
Why is the cast here needed?

Reply:
Try this type, instead of the other one, I think it will remove the need for the casts

```
type Implements<Model> = {
  [K in keyof Model]-?: z.ZodTypeAny & { _output: Model[K] };
};
```